backend/Controllers/AuthenticationController.cs

```diff
@@ -0,0 +1,35 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+using CbtBackend.Models;
+using CbtBackend.Entities;
+using CbtBackend.Services;
+
+namespace CbtBackend.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class AuthenticationController : ControllerBase {
+    private readonly ILogger<AuthenticationController> logger;
+    private readonly IUserService userService;
+
+    public AuthenticationController(IUserService userService, ILogger<AuthenticationController> logger) {
+        this.logger = logger;
+        this.userService = userService;
+    }
+
+    [AllowAnonymous]
+    [HttpPost(Name = "PostAuthentication")]
+    public IActionResult Post(UserAuthenticationRequest authenticationRequest) {
```

Comment:
According to spec `email` and `password` are sent through query params, not body

Reply:
also, according to the spec, the endpoint shouldn't return the information in format that it is returning it now, this should just return a single string with token and expiration date in `X-Expires-After` header

https://github.com/shilangyu/SE2-CBT/blob/21499498e52521ef2dcb02855c711028ca49bc4d/docs/openapi.yaml#L343

Reply:
https://github.com/shilangyu/SE2-CBT/pull/124/commits/1fc395316fa969ddc58ee8d17d9cdef9fa900d01